x86_energy_plugin.cpp

```diff
@@ -140,8 +140,8 @@ class x86_energy_measurement
 {
 public:
     x86_energy_measurement()
-    /* setting default value for reading_time */
-    : mreading_time(std::chrono::milliseconds(5)), is_thread(0)
+    /* setting default value for intervall_us */
+    : mintervall_us(std::chrono::microseconds(50000)), is_thread(0)
```

Comment:
Why the change from milliseconds to microseconds? Does that even make sense for RAPL @rschoene
IIRC RAPL has a sampling rate of 1000 Samples/seconds, but they aren't equidistant. This roughly translates to a temporal resolution of 1 millisecond, at most.

Reply:
specially if they are nor equidistant, us make sense, as you can than sample the change of the value.